src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {

```

Comment:
Using an abstract class to set behavior for classes is not recommended when interfaces are more suitable. Consider whether 'Figure' should be an abstract class or if behavior can be fully represented using interfaces.